src/Skip/Install-NugetPackage.ps1

```diff
@@ -40,38 +40,22 @@ function Install-NugetPackage
         [Parameter(Mandatory=$true)]
         [string] $OutputLocation,
         [Parameter(ParameterSetName ="FindPackage")]
-        [string] $Version,
-        [Parameter(Mandatory=$true, ParameterSetName ="InstallPackage")]
-        [NuGet.IPackage] $Package,
-        [string] $ProjectPath,
-        [bool] $RemoveFiles = $false
+        [string] $Version
     )
-    Process
-    {
-        if(-not $Package) {
-            $packages = Get-NugetPackage $PackageId $ProjectPath
+    Process {
 
-            if($Version) {
-                $packageToInstall = $packages | ? {$_.Version -eq $version}
-            }
-            else {
-                $packageToInstall = $packages | sort {$_.Version} | select -last 1
-            }
-            if(-not $packageToInstall) {
-                Write-Error "Package $packageId with version $version not found"
-            }
-        }
-        else {
-            $packageToInstall = $Package
-        }
-        $outputFileSystem = New-Object NuGet.PhysicalFileSystem $OutputLocation
-        if ($RemoveFiles) {
-            $outputFileSystem.DeleteFiles($packageToInstall.GetFiles(), $OutputLocation)
+        $nuget = ResolvePath -PackageId "NuGet.CommandLine" -RelativePath "tools\NuGet.exe"
+
+        if($Version) {
+            & $nuget install $PackageId -ExcludeVersion -PreRelease -NonInteractive -Version $Version
+        } else {
+            & $nuget install $PackageId -ExcludeVersion -PreRelease -NonInteractive        
```

Comment:
Do you also want to have the flag `-PreRelease` in case there is no specific version or would it be better to fall back to the latest stable version? Or is our versioning setup in a way that also the latest stable version is considered a prerelease?

Reply:
> **TLDR:** This should be the same behaviour as before
 
As I understood the FindPackagesById method from Nuget.Core that was used before gets all versions including prerelease.
They can then be filtered by Version/IsReleaseVersion property on the package object, which was only done in Skip if version was specified otherwise _select -last 1_ was used, which could also be a prerelease version.